scrapy/commands/crawl.py

```diff
@@ -19,6 +19,8 @@ def add_options(self, parser):
                           help="set spider argument (may be repeated)")
         parser.add_option("-o", "--output", metavar="FILE", action="append",
                           help="dump scraped items into FILE (use - for stdout)")
+        parser.add_option("-O", "--overwrite-output", metavar="FILE", action="append",
+                          help="overwrite scraped items into FILE")
```

Comment:
I wonder if it'd be more clear if help text for -o and -O is similar. E.g. (not sure about wording):

```suggestion
        parser.add_option("-o", "--output", metavar="FILE", action="append",
                          help="dump scraped items into FILE, appending them in the end (use - for stdout)")
        parser.add_option("-O", "--overwrite-output", metavar="FILE", action="append",
                          help="dump scraped items into FILE, overwrite the file")
```